app/views/draws/_oversub_report.html.erb

```diff
@@ -11,7 +11,7 @@
     </tr>
   </thead>
   <tbody>
-    <% @suite_sizes.each do |size| %>
+    <% @sizes.each do |size| %>
```

Comment:
Are we not currently sending in local variables to this partial? Fine to leave if not.